pytest.ini

```diff
@@ -6,7 +6,7 @@ log_file = pytest.log
 log_file_level = DEBUG
 log_file_format = %(asctime)s %(levelname)s %(message)s
 log_file_date_format = %Y-%m-%d %H:%M:%S
-timeout = 30
+timeout = 60
 filterwarnings =
     # These namespaces are declared in a way not conformant with PEP420. Not much we can do about that here, we should keep an eye on when this is fixed in our dependencies though.
```

Comment:
see https://github.com/xstatic-py/xstatic/issues/12

Reply:
That is unrelated to this PR and there is an issue for it already: #1919. As mentioned there it seems like the real culprit is PyFilesystem2.